DQM/DTMonitorModule/python/dtDQMOfflineSources_cff.py

```diff
@@ -14,6 +14,21 @@
                          subSystemFolder = cms.untracked.string('DT')
                          )
 
+# DT digitization and reconstruction
+# Switched to TwinMux
+from EventFilter.L1TXRawToDigi.twinMuxStage2Digis_cfi import *
```

Comment:
hi @davidlange6 
Done. Should I do the same to the equivalent 80X and 81X PR too?
Thanks